packages/type/scss/font-face/_mono.scss

```diff
@@ -11,420 +11,1156 @@
 /// @access public
 /// @group @carbon/type
 @mixin carbon--font-face-mono {
-  // .woff support for IE11
   @font-face {
     font-display: $carbon--font-display;
     font-family: 'IBM Plex Mono';
-    font-style: italic;
-    font-weight: 300;
-    src: local('IBM Plex Mono Light Italic'), local('IBMPlexMono-LightItalic'),
-      url(https://fonts.gstatic.com/s/ibmplexmono/v4/-F6sfjptAgt5VM-kVkqdyU8n1ioSflV1gMoW.woff)
+    font-style: normal;
+    font-weight: 700;
+    src: local('IBM Plex Mono Bold'), local('IBMPlexMono-Bold'),
+      url('fonts/IBM-Plex-Mono/fonts/complete/woff2/IBMPlexMono-Bold.woff2')
```

Comment:
Quick question, should these URLs be using the akamai host? Just am seeing `fonts/` at the moment

Reply:
@joshblack haha yes 🤦  I've already pushed an update for it